rewatch/src/build/clean.rs

```diff
@@ -65,7 +65,12 @@ fn clean_source_files(build_state: &BuildState, root_package: &packages::Package
         .values()
         .filter_map(|module| match &module.source_type {
             SourceType::SourceFile(source_file) => {
-                let package = build_state.packages.get(&module.package_name).unwrap();
+                let package = build_state.packages.get(&module.package_name).unwrap_or_else(|| {
+                    panic!(
```

Comment:
Is it possible to bubble this up as a `Result` instead?

Reply:
To what end, I wonder? This is a configuration error or cli argument error. Feels fine to fail her in my opinion.
We should, of course, communicate why it failed.